src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Mapping/RazorLanguageQueryEndpointTest.cs

```diff
@@ -149,7 +149,7 @@ private static RazorCodeDocument CreateCodeDocumentWithCSharpProjection(string r
             projectedCSharpSource,
             RazorCodeGenerationOptions.CreateDefault(),
             Enumerable.Empty<RazorDiagnostic>(),
-            sourceMappings,
+            sourceMappings.ToImmutableArray(),
```

Comment:
Can the enclosing method be changed to take an `ImmutableArray<SourceMapping>` to avoid the `ToImmutableArray()` here? Just wondering if you feel like threading `ImmutableArray<SourceMapping>` through the testing infrastructure. :smile:

Reply:
Does nobody read PR descriptions any more? 😛

I changed the tests to use collection expressions, but that breaks at runtime due to a roslyn issue. I then changed all the tests to use `ImmutableArray.CreateRange` and associated methods to pass in immutable arrays, and the IDE greyed out the code, and suggested using collection expressions, which break at runtime. This seemed like the safer change for now, and its only in tests.

Reply:
I did read the PR description and felt bad that you ran into the problem of collection expressions not being ready in RC1. 😉